streams/src/test/java/org/apache/kafka/streams/integration/KStreamTransformIntegrationTest.java

```diff
@@ -55,61 +60,92 @@
     private final String topic = "stream";
     private final String stateStoreName = "myTransformState";
     private final List<KeyValue<Integer, Integer>> results = new ArrayList<>();
-    private final ForeachAction<Integer, Integer> action = (key, value) -> results.add(KeyValue.pair(key, value));
+    private final ForeachAction<Integer, Integer> accumulateExpected = (key, value) -> results.add(KeyValue.pair(key, value));
     private KStream<Integer, Integer> stream;
 
     @Before
     public void before() {
         builder = new StreamsBuilder();
-        final StoreBuilder<KeyValueStore<Integer, Integer>> keyValueStoreBuilder =
-                Stores.keyValueStoreBuilder(Stores.persistentKeyValueStore(stateStoreName),
-                                            Serdes.Integer(),
-                                            Serdes.Integer());
-        builder.addStateStore(keyValueStoreBuilder);
         stream = builder.stream(topic, Consumed.with(Serdes.Integer(), Serdes.Integer()));
     }
 
+    private StoreBuilder<KeyValueStore<Integer, Integer>> storeBuilder() {
+        return Stores.keyValueStoreBuilder(Stores.persistentKeyValueStore(stateStoreName),
+            Serdes.Integer(),
+            Serdes.Integer());
+    }
+
     private void verifyResult(final List<KeyValue<Integer, Integer>> expected) {
         final ConsumerRecordFactory<Integer, Integer> recordFactory =
             new ConsumerRecordFactory<>(new IntegerSerializer(), new IntegerSerializer());
         final Properties props = StreamsTestUtils.getStreamsConfig(Serdes.Integer(), Serdes.Integer());
         try (final TopologyTestDriver driver = new TopologyTestDriver(builder.build(), props)) {
             driver.pipeInput(recordFactory.create(topic, Arrays.asList(new KeyValue<>(1, 1),
-                                                                       new KeyValue<>(2, 2),
-                                                                       new KeyValue<>(3, 3),
-                                                                       new KeyValue<>(2, 1),
-                                                                       new KeyValue<>(2, 3),
-                                                                       new KeyValue<>(1, 3))));
+                new KeyValue<>(2, 2),
```

Comment:
If you change the indention, move the first `new KeyValue(1, 1)` to it's own line.